src/Compute/Compute.Autorest/test/env.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
added these back because tests needs them to run. and other modules have these files in their test file